<launch>
    
  <node pkg="tf" type="static_transform_publisher" name="base_to_laser" 
     args="0.16 0.0 0.1 0.0 0.0 0.0 base_link front_laser_link 0.1" />

  <!-- pioneer motion driver -->
  <node pkg="p2os_driver" type="p2os" name="robot">
    <param name="port" value="/dev/ttyUSB0" />
  </node> 

  <!-- hokuyo laster scanner driver -->
  <node pkg="hokuyo_node" type="hokuyo_node" name="laser">
    <param name="frame_id" value="front_laser_link" />
    <remap from="scan" to="front_laser" />
  </node> 
</launch>
